src/main/scheduler/scheduler_tasks.c

```diff
@@ -42,7 +42,14 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .taskName = "GYRO",
         .taskFunc = taskGyro,
         .desiredPeriod = 1000000 / 1000, //Run at 1000Hz
-        .staticPriority = TASK_PRIORITY_REALTIME
+        .staticPriority = TASK_PRIORITY_REALTIME,
+    },
+
+    [TASK_ACC] = {
+        .taskName = "ACC",
+        .taskFunc = taskAcc,
+        .desiredPeriod = 1000000 / 120, //Run at 120Hz, with LPF at 15Hz this is even too much
```

Comment:
The problem is aliasing. If we have vibrations over 120Hz (which is in fact very common even for bigger machines) they will wrap down to 120Hz range - 121Hz will be wrapped down to 1Hz and pass though LPF no problem. We've been down that road before when scheduler was introduced.